Vagrantfile

```diff
@@ -74,7 +74,7 @@ Vagrant.configure(2) do |config|
      sudo apt-get update
      sudo apt-get install -y git nodejs nodejs-legacy npm python-pip python-dev postgresql-client postgresql postgresql-contrib postgresql-server-dev-all
      sudo npm install -g bower
-     sudo -u postgres psql postgres -c "CREATE USER vagrant;"
```

Comment:
When I run unit test on Vagrant, it creates test_crowdsource_dev database by using vagrant account. Without adding CREATEDB, it didn't work.